dotnet/src/Functions/Functions.UnitTests/OpenApi/OpenApiDocumentParserV30FeatureTests.cs

```diff
@@ -0,0 +1,78 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.IO;
+using System.Linq;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Microsoft.SemanticKernel.Plugins.OpenApi;
+using SemanticKernel.Functions.UnitTests.OpenApi.TestPlugins;
+using Xunit;
+
+namespace SemanticKernel.Functions.UnitTests.OpenApi;
+
+public class OpenApiDocumentParserV30FeatureTests
+{
+    /// <summary>
+    /// OpenAPI document stream.
+    /// </summary>
+    private readonly Stream _openApiDocument;
+
+    /// <summary>
+    /// System under test - an instance of OpenApiDocumentParser class.
+    /// </summary>
+    private readonly OpenApiDocumentParser _parser;
+
+    public OpenApiDocumentParserV30FeatureTests()
+    {
+        this._openApiDocument = ResourcePluginsProvider.LoadFromResource("openapi_feature_tests.json");
+        this._parser = new OpenApiDocumentParser();
+    }
+
+    [Fact]
+    public async Task ParsesAllOfAsync()
```

Comment:
nit: please use `ItCan...`, `ItShould...`, `It...` format for names where the `It` refers to the `OpenApiDocumentParser` specified in the test class name.